modules/caddyhttp/replacer.go

```diff
@@ -398,6 +403,21 @@ func getTLSPeerCert(cs *tls.ConnectionState) *x509.Certificate {
 	return cs.PeerCertificates[0]
 }
 
+type requestId struct {
+	value string
+}
+
+// Lazy generates UUID string or return cached value if present
+func (rid *requestId) String() string {
+	if rid.value != "" {
+		return rid.value
+	}
+	if id, err := uuid.NewRandom(); err == nil {
+		rid.value = id.String()
+	}
+	return rid.value
```

Comment:
```suggestion
	if rid.value == "" {
		if id, err := uuid.NewRandom(); err == nil {
			rid.value = id.String()
		}
	}
	return rid.value
```

A little more concise.